using-pagination/src/pages/products/[page].js

```diff
@@ -8,15 +8,16 @@ const limit = 1;
 const singleProductStyle = {
   display: 'flex',
   flexDirection: 'column',
-  width: '300px',
+  width: '200px',
   margin: '10px',
   color: '#000',
 };
 
 function SingleProduct({product}) {
+  const { image } = product;
   return (
     <div style={singleProductStyle} key={product.id}>
-      <img src={`https://via.placeholder.com/300x200?text=${product.name}`} width={300} height={200} alt={product.name} title={product.name} /> 
+      <img src={image.url} width={200} height={200} alt={product.name} title={product.name} /> 
```

Comment:
It's not recommended you use `<img />` if you know the width/height but instead use Next.js Image.

However, I would not show an image at all here. We have an example for that.

Reply:
@notrab we're using Next.js 9 on this project and `next/image` was introduced in v10. Unless we upgrade it, it's not possible.

Reply:
That's not really relevant though. We should aim to keep these updated regardless.